src/ui/MLFacet/MLFacetOptions.ts

```diff
@@ -44,13 +47,34 @@ export const MLFacetOptions = {
    * Specifying a value for this option is required for the `MLFacet` component to work.
    */
   field: ComponentOptions.buildFieldOption({ required: true, section: 'CommonOptions' }),
+  /**
+   * Specifies the criteria to use to sort the facet values.
+   *
+   * See {@link FacetSortCriteria} for the list and description of possible values.
+   *
+   * Default value is the `undefined`
+   */
+  sortCriteria: ComponentOptions.buildStringOption({
+    postProcessing: value => (isFacetSortCriteria(value) ? value : undefined),
+    section: 'Sorting'
+  }),
+  /**
+   * Specifies the maximum number of field values to display by default in the facet before the user
+   * clicks to show more.
+   *
+   * Minimum value is `0`.
```

Comment:
Should we set a default value as well? I remember Anne-Catherine saying she wasn't sure what the default number of displayed values should be, but it should be between 5-8? We could use 5 for now.

Reply:
Default will be on the API side, we could have our own default too it won't change much

Reply:
Same comment as above, I think it will be important to explain in detail what happens for each parameter if you leave it to default (being decided by the backend).

With that said, i think it's a good idea to leave the default up to the backend.